src/Compatibility/Maps/src/iOS/MapRenderer.cs

```diff
@@ -165,7 +164,8 @@ protected override void OnElementChanged(ElementChangedEventArgs<View> e)
 #endif
 				}
 
-				MessagingCenter.Subscribe<Map, MapSpan>(this, MoveMessageName, (s, a) => MoveToRegion(a), mapModel);
+				WeakReferenceMessenger.Default.Register<MapRenderer, MapSpan>(this, (renderer, args) => MoveToRegion(args));
```

Comment:
Same here with the closure:

```suggestion
				WeakReferenceMessenger.Default.Register<MapRenderer, MapSpan>(this, (r, args) => r.MoveToRegion(args));
```

Additionally, all these handles could be `static`, for clarity and to avoid mistakes.